features/steps/steps.rb

```diff
@@ -0,0 +1,69 @@
+Then("the Content-Type header is valid multipart form-data") do
```

Comment:
I believe some of these steps are covered in part by current maze-runner steps, and if not they should be.

New step:
- Then the request contains valid multipart form-data content

Reply:
I don't think a step for checking the content is necessary because it gets parsed by MR, so if it was invalid I'd assume the parsing would fail

This is just checking the header matches because the other assertions check against the parsed request body, which is the same for JSON requests too